src/styles/raw-themes/light-raw-theme.js

```diff
@@ -11,6 +11,7 @@ import Spacing from '../spacing';
 export default {
   spacing: Spacing,
   fontFamily: 'Roboto, sans-serif',
+  layerZIndex:20,
```

Comment:
Can we change the hierarchy?
To somethings like `zIndex.layer` since we are going to have more property?